### ISSUE TITLE
The "read with streaming" unit-test (in network_spec.js) fails on the bots when run using the `unittest` command

### ISSUE THREAD

**Issue by Snuffleupagus** [mozilla/pdf.js#7117]
As testing in PR #7116 shows, the ["read with streaming" unit-test](https://github.com/mozilla/pdf.js/blob/master/test/unit/network_spec.js#L67) fails on the bots when run using the `unittest` command. *However*, the unit-test pass when run using the `test` command.

This issue thus seems to be identical to https://github.com/mozilla/pdf.js/pull/6209#issuecomment-159606071, which means that we either need to use a locally available PDF file for that test, or change the unit-test framework to be able to deal with linked files.

/cc @brendandahl, @yurydelendik

**Comment by Snuffleupagus** [comment]
I've just submitted PR https://github.com/mozilla/botio-files-pdfjs/pull/16 which should, once pushed to the bots, allow us to reference already downloaded linked PDF files in unit-tests.

Even with the bot patch, there would be one remaining issue. It's still required that e.g. `browsertest` is run in order for linked PDF files to actually be downloaded. Hence I've also submitted https://github.com/mozilla/pdf.js/pull/7122, to make the `unittest` command able to handle linked PDF files as well.
